handlers/search.go

```diff
@@ -164,7 +170,7 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 		if parseErr == nil { // search the validator by its index
 			err = db.ReaderDb.Select(result, `SELECT validatorindex AS index, pubkeyhex as pubkey FROM validators WHERE validatorindex = $1`, indexNumeric)
 		} else if thresholdHexLikeRE.MatchString(search) {
```

Comment:
You have to use `strippedSearch` here. Otherwise, searching for a validator public key (example: `0x80002a650ac64c6c6076f0186e49a8a1e3312d3c4015dcc8d1374afb6fe3df5e7e96203a193770e134b33953e9e469a1`) does not work.